docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -198,22 +198,49 @@ streams:
           saturation_threshold: 0.95
   ...
 
-It configures the overload manager to reset certain streams depending on the
-heap size.  When the heap usage is less than 85%, no streams will be reset.
-When heap usage is at or above 85%, we start to reset certain memory classes
-(e.g. streams using memory within a power of two range). There are 8 buckets,
-with the last bucket capturing all of the streams using :math:`>= 128 *
-minimum_threshold_for_tracking`.  In this example the `minimum_threshold_for_tracking` is 1MB.
+By setting the `minimum_account_to_track_power_of_two` to `20`, we will only track
```

Comment:
Could you spell out how 20 translates to 1MB? I assume it's due to 2^20 but would be helpful to be explicit in the docs

Reply:
Done